includes/class-wc-payments-webhook-processing-service.php

```diff
@@ -371,7 +374,7 @@ private function process_webhook_payment_intent_failed( $event_body ) {
 
 		// Get the order and make sure it is an order and the payment methods match.
 		$order             = $this->get_order_from_event_body_intent_id( $event_body );
-		$payment_method_id = $charges_data['payment_method'] ?? null;
+		$payment_method_id = $payment_method['id'] ?? null;
```

Comment:
Same with my above comment. I don't understand this change.

I see that originally the value of `$payment_method_id` is from `data.object.charges.data[0].payment_method` and now it's from `data.object.last_payment_error.id`.